workspace/src/localization/localization_py/localization_py/dynamics.py

```diff
@@ -0,0 +1,91 @@
+import numpy as np
+import yaml
+import math
+class Dynamics:
+    """A 4DOF dynamics implementation.
+    
+    This currently uses the 4DOF motion model. The Motion Model inputs are throttle and steering, and the observation model uses GPS and Magnetometer as sensors. States are cartesian coordinates x, y, heading theta, and velocity v.
+
+    Attributes:
+        c_1: The motor resistence torque linear component.
+        c_0: The motor resistence torque constant component.
+        l: The length of the vehicle.
+        r_wheel: The radius of the wheel.
+        i_wheel: the intertia of the wheel.
+        gamma: The gear ratio.
+        tau_0: The stalling torque for the motor torque model.
+        omega_0: The maximum no-load speed for the motor torque model.
+        df_1_dv: The derivative of the scaled motor torque relative to the velocity of the vehicle.
+    """
+    def __init__(self, dt, dynamics):
```

Comment:
Maybe  a better name here instead of "dyanmics"? 
Maybe "params"? or "vehicle_params"